tests/cases/fourslash/goToDefinitionVariableAssignment.ts

```diff
@@ -4,9 +4,9 @@
 // @checkJs: true
 // @filename: foo.js
 ////const Bar;
-////const /*def1*/Foo = /*def2*/Bar = function () {}
+////const Foo = /*def*/Bar = function () {}
```

Comment:
I am not sure this is correct. I would expect this to go to `function () {}` instead.

Reply:
I noticed that too, but this is how it was before, and it's not in the code that I changed, so unrelated to the fix...  (And BTW, there's some special treatment for `function` there which is doing it -- it would get the function if it was an arrow function.)

Reply:
But its changing behavior of getting one definition instead of two.. It would be confusing to go to Bar here and not Foo when doing new Foo() ?

Reply:
It's "half of a change", since already it *did* point at `Bar =` before.

In any case, I agree that it would be better to point at the expression instead, but that would be a change in a different place in the code.  There is one part that finds the value, and then a different part checks whether to show just the value or also the declared names.  I changed the latter, and what you're talking about is in the former which I didn't change.  So that should be a different PR, I think.

Reply:
I'll open a new issue for this.